mobify/dustjs
# Dust components proof of concept

Can we make dust work like this: https://gist.github.com/ry5n/1455a1dbf2b6ff56fcfd

Status: **Do not merge thx**
## Changes
- Adds a new feature to dust, here called “include” but which ultimately I think should look more like the syntax in the Gist linked above)
## How to test-drive this PR
- Check out this branch
- Run `node toy/` from the project root
- Review the console output (this is a poor man’s test suite for the features outlined in `toy/templates/include.dust`).
## Research resources
- Ember Components is similar: http://emberjs.com/guides/components/
